### PR TITLE
Improve Wedge3D documentation of halves_to_use

### LINKED ISSUE #795: Improve Wedge3D documentation of 'halves_to_use'.
In Wedge3D there is a parameter `halves_to_use` passed into the constructor,
and a corresponding enum class `WedgeHalves`.  The documentation says that this parameter determines whether to "use" logical coordinates xi in [0,1] or [-1,0], but I don't know what "use" means, and this documentation does not make it clear what this parameter does.  In particular:

  - Does this apply only for the xi direction, and not the eta direction?  (this seems to be the case).
  - Do the logical coordinates xi still go from [-1,1] if `halves_to_use` is not 'Both'? In other words, if I had an arbitrary point inside the domain and I inverse-mapped it through Wedge3D, do I get a logical point that is in [-1,1] (as opposed to [-1,0] or [0,1])? (This seems to be the case from looking through the code, but this seems to contradict the documentation).
  - It seems (from looking through the code) that if this parameter is not 'Both', the resulting shape has a face normal to the x direction (assuming default OrientationMap).  Is this the case?  It should be described in the documentation.

### Component:

- [ ] Code
- [x] Documentation
- [ ] Build system
- [ ] Continuous integration

### Desired feature:

- [ ] Detail 1
- [ ] Detail 2
- [ ] Detail 3

### PR DESCRIPTION
**Pull request by mar-celine**
## Proposed changes
Closes #795 

### Types of changes:

- [ ] Bugfix
- [ ] New feature

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests, `clang-tidy` and `IWYU`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

**Comment by nilsdeppe** [comment]
If this closes an issue could you please update the comment to say `closes #...` so that GitHub automatically closes the issue when this gets merged?

**Comment by wthrowe** [comment]
Doesn't render correctly.

**Review by wthrowe — DISMISSED** [review]

**Comment by nilsdeppe** [comment]
@markscheel since you're the author of the issue this fixes would you mind being the second reviewer on this?

**Comment by kidder** [comment]
@markscheel please look at

File: src/Domain/CoordinateMaps/Wedge3D.hpp [hunk 1]
```diff
@@ -240,11 +240,17 @@ class Wedge3D {
...
-   * currently only useful in constructing domains for binary systems.
+   * \param halves_to_use Determines whether to split the Wedge3D in half. If
+   * the wedge is split in half, the resulting shape has a face normal to the x
+   * direction (assuming default OrientationMap). If the wedge is split in half,
+   * an intermediate affine map is applied to the logical xi coordinate such
+   * that the interval [-1,1] is mapped to the corresponding logical half of the
+   * wedge. For example, if `UpperOnly` is specified, [-1,1] is mapped to
+   * [0,1], and if `LowerOnly` is specified, [-1,1] is mapped to [-1,0]. In the
+   * case of `Both`, no intermediate map is applied. In all cases, the logical
+   * points returned from the inverse map function call lie in the range [-1,1]
```
Comment (markscheel) [inline src/Domain/CoordinateMaps/Wedge3D.hpp]: Minor: please reword to "...points returned by the inverse map will lie in the range..."

File: src/Domain/CoordinateMaps/Wedge3D.hpp [hunk 2]
```diff
@@ -240,11 +240,17 @@ class Wedge3D {
...
-   * full wedge, or the full wedge entirely (value of `Both`). Half wedges are
-   * currently only useful in constructing domains for binary systems.
+   * \param halves_to_use Determines whether to split the Wedge3D in half. If
+   * the wedge is split in half, the resulting shape has a face normal to the x
+   * direction (assuming default OrientationMap). If the wedge is split in half,
+   * an intermediate affine map is applied to the logical xi coordinate such
+   * that the interval [-1,1] is mapped to the corresponding logical half of the
+   * wedge. For example, if `UpperOnly` is specified, [-1,1] is mapped to
+   * [0,1], and if `LowerOnly` is specified, [-1,1] is mapped to [-1,0]. In the
+   * case of `Both`, no intermediate map is applied. In all cases, the logical
```
Comment (markscheel) [inline src/Domain/CoordinateMaps/Wedge3D.hpp]: How about change to "The case of `Both` means a full wedge, with no intermediate map applied."

File: src/Domain/CoordinateMaps/Wedge3D.hpp [hunk 3]
```diff
@@ -240,11 +240,17 @@ class Wedge3D {
    * somewhere in between
    * \param with_equiangular_map Determines whether to apply a tangent function
    * mapping to the logical coordinates (for `true`) or not (for `false`).
-   * \param halves_to_use Determines whether to use the logical xi
-   * coordinates in the [0,1] interval (value of `UpperOnly`) of the full wedge,
-   * the coordinates in the [-1,0] interval (value of `LowerOnly`) of the
-   * full wedge, or the full wedge entirely (value of `Both`). Half wedges are
-   * currently only useful in constructing domains for binary systems.
+   * \param halves_to_use Determines whether to split the Wedge3D in half. If
```
Comment (markscheel) [inline src/Domain/CoordinateMaps/Wedge3D.hpp]: How about "split the Wedge3D in half" -> "construct a full wedge or only half a wedge."

And similarly below in two places "If the wedge is split in half" -> "If constructing only half a wedge".

(reasoning: "split a wedge in half" sounds like you are constructing two halves, but this class constructs only one half).

**Review by markscheel — CHANGES_REQUESTED** [review]
Only minor changes.  Much much better than previous documentation!

**Comment by kidder** [comment]
@markscheel please look at

File: src/Domain/CoordinateMaps/Wedge3D.hpp [hunk 4]
```diff
@@ -240,10 +240,16 @@ class Wedge3D {
...
    * \param with_equiangular_map Determines whether to apply a tangent function
    * mapping to the logical coordinates (for `true`) or not (for `false`).
-   * \param halves_to_use Determines whether to use the logical xi
-   * coordinates in the [0,1] interval (value of `UpperOnly`) of the full wedge,
-   * the coordinates in the [-1,0] interval (value of `LowerOnly`) of the
-   * full wedge, or the full wedge entirely (value of `Both`). Half wedges are
+   * \param halves_to_use Determines whether to construct a full wedge or only
+   * half a wedge. If constructing only half a wedge, the resulting shape has a
+   * face normal to the x direction (assuming default OrientationMap). If the
+   * wedge is split in half, an intermediate affine map is applied to the
```
Comment (markscheel) [inline src/Domain/CoordinateMaps/Wedge3D.hpp]: "the wedge is split in half" -> "constructing half a wedge".

**Review by markscheel — COMMENTED** [review]
One small additional change; please squash.

**Review by markscheel — DISMISSED** [review]

**Comment by nilsdeppe** [comment]
@gsb76 please rebase this on the latest develop so travis will build.

**Review by kidder — APPROVED** [review]

**Review by nilsdeppe — APPROVED** [review]

**Pull request merged**